### DISCUSSION TITLE
Sometimes will appear error with your starter

### DISCUSSION THREAD

**Discussion by SCUseLGY** [majusko/pulsar-java-spring-boot-starter#238] [Show and tell]
Mr.majusko,
when I use spring-boot-starter-parent which version is 2.2.6-RELEASE，and import your starter which version is 1.1.2，Springboot Application run failed.

ErrorInfo:
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'consumerAggregator' defined in URL [jar:file:/Users/.m2/repository/io/github/majusko/pulsar-java-spring-boot-starter/1.1.2/pulsar-java-spring-boot-starter-1.1.2.jar!/io/github/majusko/pulsar/consumer/ConsumerAggregator.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.github.majusko.pulsar.consumer.ConsumerAggregator]: Constructor threw exception; nested exception is java.lang.NoClassDefFoundError: reactor/core/publisher/Sinks
	

After review the External Libraries,I found that reactor-core version is 3.3.3 which covered the version in your starter pom.

**Comment by majusko** [comment]
Hi @SCUseLGY, yes, there was a depreciation of Emitters in the reactor core library, and we refactored the Emitters to Sinks as recommended by the rector core. I would recommend using the later version of spring boot. This issue was also already discussed in #150

**Reply by SCUseLGY** [reply]
roger，thanks
